src/browser/findDOMNode.js

```diff
@@ -36,8 +34,8 @@ function findDOMNode(componentOrElement) {
     return ReactMount.getNodeFromInstance(componentOrElement);
   }
   invariant(
-    !(componentOrElement.render != null && typeof(componentOrElement.render) === 'function'),
-    'Component contains `render` method but is not mounted in the DOM',
+    componentOrElement.render == null || typeof(componentOrElement.render) !== 'function',
```

Comment:
I think if most of the times in the source we see `invariant(condition)`, it's better to have it that way without negations. I think demorganized version reads more easily.